### PR TITLE
Pablo designs

### PR DESCRIPTION
**Pull request by palm9727**
Just opening a pull request to see the things I need to change.

### CODE REVIEW

**Review by reillymck — CHANGES_REQUESTED** [review]

File: scripts/bfasst/netlist_mapping/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+* The [functional_mapping.py](./functional_mapping.py) script will map two different netlists based on the trees generated from the functions of the LUTs before each flipflop.
+* The [ccl_mapping.py](./ccl_mapping.py) script will map two different netlists using structural algorithms for the flipflops connected to carry-chains and shift-registers, 
+and a functional algorithm for the rest of the flipflops.
+
+* Here is an example of how to run one of the scripts:  
+$ python ccl_mapping.py netlist_examples/tx_impl.v netlist_examples/tx_reversed.v > mapped_points.txt
+
+* The [mapped_points.txt](./mapped_points.txt) file generated can be used in Conformal Equivalence Checker by adding the following lines to the compare.do file:  
+set system mode lec -nomap  
+REAd MApped Points mapped_points.txt
```
[the remark this answers is not in the document]
Comment (palm9727) [reply scripts/bfasst/netlist_mapping/README.md:12]: Yes

File: scripts/bfasst/netlist_mapping/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# ccl_netlist_mapping
+* The [structural_mapping.py](./structural_mapping.py) script will map two different netlists based on the structure of each block and its connections.
+* The [functional_mapping.py](./functional_mapping.py) script will map two different netlists based on the trees generated from the functions of the LUTs before each flipflop.
+* The [ccl_mapping.py](./ccl_mapping.py) script will map two different netlists using structural algorithms for the flipflops connected to carry-chains and shift-registers, 
+and a functional algorithm for the rest of the flipflops.
+
+* Here is an example of how to run one of the scripts:  
+$ python ccl_mapping.py netlist_examples/tx_impl.v netlist_examples/tx_reversed.v > mapped_points.txt
```
[the remark this answers is not in the document]
Comment (palm9727) [reply scripts/bfasst/netlist_mapping/README.md:8]: Done

File: scripts/bfasst/netlist_mapping/functional/qm.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
```
[the remark this answers is not in the document]
Comment (palm9727) [reply scripts/bfasst/netlist_mapping/functional/qm.py:1]: Most likely. I just didn't know how to do it so I added qm.py as part of my code.

File: scripts/bfasst/netlist_mapping/functional/converter.py [hunk 4]
```diff
@@ -0,0 +1,186 @@
+"""This file contains functions used to transform numbers between hex and binary"""
```
[the remark this answers is not in the document]
Comment (palm9727) [reply scripts/bfasst/netlist_mapping/functional/converter.py:1]: I just though it would be faster to build the functions that do exactly what I wanted.

**Comment by jgoeders** [comment]
@palm9727 @reillymck Is this ready to be merged now?

**Pull request merged**